bcrembed/__main__.py

```diff
@@ -32,33 +32,33 @@
 stdout = Console()
 
 @app.command()
-def antiberty(inpath: str, colname: str, outpath: str):
+def antiberty(inpath: str, sequence_input: str, outpath: str):
     """
     Embeds sequences using the AntiBERTy model.
 
     Args:
         inpath (str): The path to the input file. The file should be in AIRR format.
-        colname (str): The name of the column in the input file that contains the sequences to be embedded.
+        sequence_input (str): Input sequences (H for heavy chain, L for light chain, HL for heavy and light concatenated). 
         outpath (str): The path where the embeddings will be saved.
 
     Usage:
-        bcrembed antiberty tests/AIRR_rearrangement_translated.tsv HL out.pt
+        bcrembed antiberty tests/AIRR_rearrangement_translated_single-cell.tsv HL out.pt
 
     Note:
         This function prints the number of sequences being embedded, the batch number during the embedding process, 
         the time taken for the embedding, and the location where the embeddings are saved.
     """
     
-    dat = pivot_airr(inpath) # H, L, HL
+    dat = process_airr(inpath, sequence_input)
     stdout.print(f"Embedding {dat.shape[0]} sequences using antiberty...")
     max_length = 512-2
-    X = dat.loc[:,colname]
+    X = dat.loc[:, "sequence_vdj_aa"]
```

Comment:
I would still let the user specify which translated column do they want to use for embedding, but set the `sequence_vdj_aa` as default